### PR TITLE
Make entity, migrations and modify relatives backend files

### PR DESCRIPTION
**Pull request by Raphael-Marchese**
# Description

Please include here a summary of the changes and the related issue.

# Changes

| Q             | A        
|---------------| ---------
| Issue         | #32 ..
| Type          | <ul><li>- [x] Feat</li><li>- [ ] Hotfix</li><li>- [ ] Doc</li><li>- [ ] Refacto</li></ul>
| Break changes |   No

### CODE REVIEW

File: api/src/Entity/Repairer.php [hunk 1]
```diff
@@ -89,8 +89,8 @@
     'country' => 'ipartial',
     'bikeTypesSupported.id' => 'exact',
     'bikeTypesSupported.name' => 'ipartial',
-    'repairerType.id' => 'exact',
-    'repairerType.name' => 'ipartial',
+    'repairerTypes.id' => 'exact',
+    'repairerTypes.name' => 'ipartial',
```
Comment (clementtalleu) [inline api/src/Entity/Repairer.php:93]: of course, same here

File: api/src/Entity/Repairer.php [hunk 2]
```diff
@@ -115,10 +115,12 @@ class Repairer
     #[Groups([self::REPAIRER_READ, self::REPAIRER_WRITE, self::REPAIRER_COLLECTION_READ])]
     public ?User $owner = null;
 
-    #[ORM\ManyToOne]
-    #[ORM\JoinColumn(nullable: true)]
+    #[ORM\JoinTable(name: 'repairer_repairer_type')]
```
Comment (clementtalleu) [inline api/src/Entity/Repairer.php:118]: Do we need the jointTable ? Doctrine could not detect it automatically ? Do not remember if we add this attribute for others manyToMany in the project

**Review by clementtalleu — CHANGES_REQUESTED** [review]

File: api/src/Entity/Repairer.php [hunk 3]
```diff
@@ -89,8 +89,8 @@
     'country' => 'ipartial',
     'bikeTypesSupported.id' => 'exact',
     'bikeTypesSupported.name' => 'ipartial',
-    'repairerType.id' => 'exact',
-    'repairerType.name' => 'ipartial',
+    'repairerTypes.id' => 'exact',
```
[the remark this answers is not in the document]
Comment (Raphael-Marchese) [reply api/src/Entity/Repairer.php:92]: Yes it works, api-platform allow that, We did it before with the bikeTypeSupported collection

**Review by davy-beauzil — COMMENTED** [review]
I tested the front end and it seems to work fine, good work 👍 
Just the step, when as user, we take an appointment with a repairer who is an itinerant repairer, we request directly the address of appointment, but I think we should to wait Cécile to know the best solution to implement.
Else, there is a front-end test broken to fix for the CI.

**Review by davy-beauzil — APPROVED** [review]

**Pull request merged**